src/components/ProgramCard.tsx

```diff
@@ -146,13 +146,23 @@ const ProgramCard: FC<ProgramCardProps> = ({
           )}
         </Stack>
 
-        <Typography
-          typography="subheading2"
-          color="warning100.main"
-          minHeight="26px"
-        >
-          {program.ctPrice}
-        </Typography>
+        <Stack>
+          <Typography
+            typography="captionregular"
+            color="neutral400.main"
+            mb="4px"
+          >
+            {program.slug.includes('english') ? 'Starting From' : 'Mulai Dari'}
```

Comment:
```suggestion
            {program.slug.includes('english') ? 'Starting From' : 'Mulai Dari'}
```
jangan by slug tapi by language posible?

Reply:
Oke kakk